config.py

```
@@ -3496,3 +3500,149 @@ def on_reset_all_data(self):
+        if not self.gui:
+            QMessageBox.warning(self, 
+                self.i18n.get('error', 'Error'),
+                self.i18n.get('library_no_gui', 'GUI not available'))
+            return
+        
+        from .utils import update_library_metadata
+        
+        # 显示处理中提示
+        self.update_button.setEnabled(False)
+        self.update_button.setText(self.i18n.get('library_updating', 'Updating...'))
+        QApplication.processEvents()
```

Ruff (0.14.13)

Undefined name `QApplication`

(F821)